allennlp_models/generation/models/simple_seq2seq.py

```diff
@@ -45,7 +43,11 @@ class SimpleSeq2Seq(Model):
     target_embedding_dim : `int`, optional (default = `'source_embedding_dim'`)
         You can specify an embedding dimensionality for the target side. If not, we'll use the same
         value as the source embedder's.
-    attention : `Attention`, optional (default = `None`)
+    target_pretrain_file : `str` (default = None)
```

Comment:
```suggestion
    target_pretrain_file : `str`, optional (default = None)
```

Reply:
Why did you resolve this? Isn't that parameter indeed optional?

I guess I would prefer it if the parameter was marked as `Optional[str]` too, but we don't do that consistently in the codebase yet, so I certainly don't expect you to fix it as part of this.

Reply:
Sorry, I don't understand what you mean. The `target_pretrain_file ` parameter is already optional and I just adjust the note part to pass the doc test.

Reply:
@epwalsh suggested a change to mark the parameter as optional in the docs as well. It looks like you didn't take that suggestion? Or did I look at the wrong file?

Reply:
Sorry, it looks like I looked at the wrong file. This change is already in.